keel-sql/src/main/resources/db/changelog/20210114-verification-state-task-list.yml

```diff
@@ -0,0 +1,11 @@
+databaseChangeLog:
+- changeSet:
+    id: update-verification-status-enum
+    author: fletch
+    changes:
+    - sql:
+        sql: |-
+          update verification_state
+          set metadata = json_object(
+            'tasks', json_array(metadata -> '$.taskId')
+          )
```

Comment:
Migrates any existing database entries to the new format

Reply:
Any chance there might be any records without a `taskId`?

Reply:
Hmm, there aren't at the moment but I guess it's possible at the time it runs. I'll revise this

Reply:
Ok, I added `where metadata -> '$.taskId' is not null`. I don't think we ever insert to that table until we have launched the Orca task, but it doesn't hurt to be extra careful.